### PR TITLE
add show user guide link to help

### LINKED ISSUE #1963: User Guides are not accessible from the app
### Description

We have a collection of User Guides that are accessible from the main Desktop site, but no Help menu item to take the user to them exists in the app:

### Version

**GitHub Desktop version:** 0.6.0

**OS version:** macOS Sierra 10.12.5

### Steps to Reproduce

1. Launch the application.
1. Feel unclear about something.
1. Click the Help menu.

**Expected behaviour:** see a menu item which links to the User Guides: https://help.github.com/desktop-beta/guides/

**Actual behaviour:** no helpful menu item 😢 

**Reproduces how often:** 100%

### PR DESCRIPTION
**Pull request by shimpossible**
Adds "user guide" link to fix #1963

### CODE REVIEW

**Comment by lee-dohm** [comment]
Built this locally. Looks good and functioned :+1:

I'm not sure about the build failure though, so I'll let someone else give the final approval.

**Comment by shimpossible** [comment]
yeah, I'm wondering about the failure too.  Can't seem to figure out what caused it

**Comment by shimpossible** [comment]
As far as i can tell the .electron cache in appveyor is the only difference in the working and not working builds.

**Comment by shiftkey** [comment]
> Not sure why 1.6.10 is, and that is the one causing the error.

The `electron` package has recently started bundling it's own TS declarations, which make `@types/electron` redundant. However we haven't migrated yet due to a CI issue with running `electron-mocha` - see #1430 for that progress.

**Comment by shiftkey** [comment]
Once #1976 has landed you can merge `master` into this branch and it'll be green again.

File: app/src/main-process/menu/build-default-menu.ts [hunk 1]
```diff
@@ -286,6 +286,13 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
     },
   }
 
+  const showUserGuide: Electron.MenuItemOptions = {
+    label: 'Show User Guide',
```
Comment (shiftkey) [inline app/src/main-process/menu/build-default-menu.ts]: The link mentions "GitHub Desktop User Guide**s**" - I think the text here should also be pluralized.

**Pull request merged**